sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -45,20 +47,23 @@ set -e
 date
 hostname
 echo ${SLURM_JOBID} ${SLURM_ARRAY_TASK_ID}
-# output_path = output_path passed to Job objects + 'NuQCJob'
-# e.g.: working-directory/ConvertJob, working-directory/QCJob...
-cd {{output_path}}
+### output_path = output_path passed to Job objects + 'NuQCJob'
+### e.g.: working-directory/ConvertJob, working-directory/QCJob...
+cd {{output_path}}/tmp
```

Comment:
this is different from TMPDIR?

Reply:
Apparently so? This change was needed for the new code to work within SPP. I'll review TMPDIR and see if I can resolve it.